docs/integration/linux-integration.md

```diff
@@ -0,0 +1,53 @@
+
+Sematext Linux Monitoring provides insights into your Linux infrastructure, automatically monitoring servers, processes, packages and more across various Linux distributions. Check the following areas for more information:
+
+- [Linux Servers](https://sematext.com/docs/monitoring/servers/)
+- [Linux Processes](https://sematext.com/docs/monitoring/processes/)
+- [Linux Packages](https://sematext.com/docs/monitoring/inventory/)
+- [Events](https://sematext.com/docs/events/), etc.
+
+## How it works
+
+A lightweight agent, called [Sematext Agent](../agents/sematext-agent), will be installed on your Linux machine. This agent sends all infrastructure data to an Infra App in Sematext Cloud, whose token is configured during the agent's installation.
+
+The agent itself can be installed either using a [native package manager](https://sematext.com/docs/agents/sematext-agent/installation/) for each distribution or using configuration automation tools like Ansible. Every option is available together with instructions during the [creation of an Infra App](https://sematext.com/docs/monitoring/infrastructure/#create-an-infra-app).
+
+## Supported Linux distributions
+
+Linux Monitoring supports the following Linux distributions:
+
+- Ubuntu
+- Debian
+- CentOS
+- Red Hat Enterprise Linux (RHEL)
+- Fedora
+- openSUSE
+- Amazon Linux
+
+For more information, please refer to our [platform support policy](https://sematext.com/docs/agents/sematext-agent/platform-support-policy/#linux) page.
+
+## Infrastructure Reports
+
+As soon as the Sematext Agent is installed, you can navigate to Sematext Cloud > Infrastructure section and access a wide variety of prebuilt reports. These reports contain hundreds of live [infrastructure metrics](https://sematext.com/docs/agents/sematext-agent/os-metrics/) together with their historical data.
+
+![Infra App metrics](../images/monitoring/infra-metrics.gif)
+
+All the metrics can be filtered or grouped by each individual host among other [supported tags](https://sematext.com/docs/tags/).
+
+Moreover, some reports contain useful information about the [processes](https://sematext.com/docs/monitoring/processes/) running on your Linux servers 
+
+![Infrastructure Processes](../images/monitoring/infra-processes.gif)
+
+Other reports give you insights about machine-related information such as host, VM, or container properties, kernel versions, and installed packages. All these are part of [Inventory Monitoring](https://sematext.com/docs/monitoring/inventory/).
+
+![Infrastructure Inventory](../images/monitoring/infra-inventory.gif)
+
+## Out of the box alerts
+
+Each time you create a new Infra App in Sematext Cloud to monitor your Linux servers, a number of pre-configured alerts are created as well, fine tuned to catch various issues and prevent your systems to go down. 
+
+Refer to our [infrastructure default alerts](https://sematext.com/docs/monitoring/servers/#core-infrastructure-alerting) to learn more information.
```

Comment:
@cpipilas would it make more sense to instead move that section here since all ARs listed there are specifically for Linux and I assume on WM page there will also be an `Out of the box alerts` section for default Win ARs.

Reply:
Most of these alerts are common to both Windows & Linux envs. I've now updated them to label those that are platform specific and I also added a ref in these core ARs in Windows integration as well.